keying.go

```diff
@@ -0,0 +1,47 @@
+package srtp
+
+// KeyingMaterialExporter allows package SRTP to extract keying material
+type KeyingMaterialExporter interface {
+	// TODO: To match crypto/tls label should be a string.
+	ExportKeyingMaterial(label []byte, context []byte, length int) ([]byte, error)
```

Comment:
Bleh, mind fixing this TODO? Even if you don't update `DTLS` we can have a little wrapper that does `[]byte -> string` before calling `ExportKeyingMaterial` I am afraid little tech debt like this will make people ignore `pions/dtls`